src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/CohortControllerTests/WhenGettingCohorts.cs

```diff
@@ -17,49 +17,72 @@ public class WhenGettingCohorts
         [Test]
         public async Task ThenCallsModelMapper()
         {
-            var fixture = new WhenGettingCohortsFixture();
+            var f = new WhenGettingCohortsFixture();
 
-            await fixture.Act();
+            await f.Sut.Cohorts(f.Request);
 
-            fixture.VerifyMapperWasCalled();
+            f.ModelMapperMock.Verify(x => x.Map<CohortsViewModel>(f.Request));
         }
 
         [Test]
-        public async Task ThenReturnsView()
+        public async Task ThenReturnsCohortsViewModel()
         {
-            var fixture = new WhenGettingCohortsFixture();
+            var f = new WhenGettingCohortsFixture();
 
-            var result = await fixture.Act() as ViewResult;
+            var result = await f.Sut.Cohorts(f.Request) as ViewResult;
 
             Assert.NotNull(result);
             Assert.AreEqual(typeof(CohortsViewModel), result.Model.GetType());
```

Comment:
Probably the extension method is more readable, but not fussed about it. Leave it to you if you want to change.
result.
VerifyReturnsViewModel().WithModel<CohortsViewModel>();